components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -112,7 +112,13 @@ ext.gleanGenerateMetricsAPI = {
     variant ->
 
     def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
-    def fullNamespace = "${variant.applicationId}.GleanMetrics"
+    def applicationId = variant.applicationId
```

Comment:
nit: I'd go for something like `filteredApplicationId`. Looks like `applicationId` is already a variable in this context (I only noticed because I mistakenly removed the `def` while testing this locally. Of course the IDE won't tell us :E)